src/middleware/index.js

```diff
@@ -48,15 +49,64 @@ if (process.env.MIXPANEL_TOKEN) {
 }
 
 const analyticsMiddleware = store => next => action => {
+    let createEvents = ['CREATE_NEW_ACCOUNT', 'CREATE_ACCOUNT_WITH_SEED_PHRASE','CHECK_ACCOUNT_AVAILABLE', 'CHECK_NEW_ACCOUNT' ]
+    let twoFAEvents = ['GET_2FA_METHOD', 'INIT_TWO_FACTOR', 'VERIFY_TWO_FACTOR', 'PROMPT_TWO_FACTOR', 'RESEND_TWO_FACTOR']
+    let ledgerEvents = ['GET_LEDGER_KEY', 'GET_LEDGER_PUBLIC_KEY', 'ADD_LEDGER_ACCESS_KEY', 'CONNECT_LEDGER', 'DISABLE_LEDGER',
+'GET_LEDGER_ACCOUNT_IDS', 'ADD_LEDGER_ACCOUNT_ID', 'SAVE_AND_SELECT_LEDGER_ACCOUNTS']
+    let accesskeyEvents = ['GET_ACCESS_KEYS', 'REMOVE_ACCESS_KEY','REMOVE_NON_LEDGER_ACCESS_KEYS', 'ADD_ACCESS_KEY', 'ADD_ACCESS_KEY_SEED_PHRASE']
+    let recoveryEvents = ['DELETE_RECOVERY_METHOD', 'RECOVER_ACCOUNT_SEED_PHRASE', 'SETUP_RECOVERY_MESSAGE_NEW_ACCOUNT']
+    let transactionEvents = ['DEPLOY_MULTISIG', 'CHECK_NEAR_DROP_BALANCE', 'SET_LEDGER_TX_SIGNED','SIGN_AND_SEND_TRANSACTIONS', 'SEND_MONEY', 'SWITCH_ACCOUNT']
+    let stakeEvents = ['UPDATE_STAKING', 'STAKE', 'UNSTAKE','WITHDRAW']
+    let trackingEvents = createEvents.concat(twoFAEvents).concat(ledgerEvents).concat(accesskeyEvents).concat(recoveryEvents).concat(transactionEvents).concat(stakeEvents)
```

Comment:
@vgrichina  These events are all what we need and see this line, I include all events.

Reply:
@icerove there are no arguments from these events that will get sent to mixpanel correct?

e.g. RECOVER_ACCOUNT_SEED_PHRASE should NOT send any information about seed phrase to mixpanel.

Reply:
this will not seed any extra information to mixpanel unless write down